templates/compose/wakapi.yml

```diff
@@ -0,0 +1,51 @@
+# documentation: https://github.com/muety/wakapi#%EF%B8%8F-how-to-use
+# slogan: Keep Track of Your Coding Time.
+# tags: wakatime,track,time
+# logo: svgs/wakapi.svg
+
+services:
+  wakapi:
+    build: .
+    ports:
+      - 3000:3000
+    restart: unless-stopped
+    environment:
+      # See README.md and config.default.yml for all config options
+      - SERVICE_FQDN_WAKAPI
+      - SERVICE_WAKAPI_DB_TYPE="postgres"
+      - WAKAPI_DB_NAME=$POSTGRESQL_DATABASE
+      - WAKAPI_DB_USER=$SERVICE_USER_POSTGRESQL
+      - WAKAPI_DB_HOST="db"
+      - WAKAPI_DB_PORT="5432"
+      - WAKAPI_DB_PASSWORD_FILE="/run/secrets/db_password"       # alternatively, set WAKAPI_DB_PASSWORD directly without the use of secrets
+      - WAKAPI_PASSWORD_SALT_FILE="/run/secrets/password_salt"   # alternatively, set WAKAPI_PASSWORD_SALT directly without the use of secrets
+      - WAKAPI_MAIL_SMTP_PASS_FILE="/run/secrets/smtp_pass"      # alternatively, set WAKAPI_MAIL_SMTP_PASS directly without the use of secrets
+    secrets:
+      - password_salt
+      - smtp_pass
+      - db_password
+
+  db:
```

Comment:
please look at how other services implement Postgres and use that logic not this one:
https://github.com/coollabsio/coolify/blob/dc92fe8760ab4929abe089a57e46c2d213c44104/templates/compose/nextcloud-with-postgres.yaml#L35-L47